tests/Core/Slider/FluentSliderTests.razor

```diff
@@ -38,7 +38,7 @@
     {
         double value = 3.5;
         // Arrange
-        var cut = Render(@<FluentSlider Min="-1.50" Max="7.0" Step="0.5" @bind-bind-Value="@value"></FluentSlider>);
+        var cut = Render(@<FluentSlider Min="-1.50" Max="7.0" Step="0.5" @bind-Value="@value" Label="label"></FluentSlider>);
```

Comment:
Weird, before the tests approved `value="0"` and not 3.5. Any explanation?

Reply:
I think that has to do with the `@bind-bind-Value` that was there before